aptos-move/framework/supra-framework/sources/genesis.move

```
@@ -195,7 +194,6 @@ module supra_framework::genesis {
         reconfiguration::initialize(&supra_framework_account);
         block::initialize(&supra_framework_account, epoch_interval_microsecs);
         state_storage::initialize(&supra_framework_account);
-        automation_registry::initialize(&supra_framework_account, epoch_interval_microsecs);
```

until we have initialization flow properly implemented can you please have `automation_registry::initializate_by_default` function which will be called here and will utilize some hard-coded default values. Otherwise tests will fail and testing can not be done.